### PR TITLE
Add total playtime to the stats page

### LINKED ISSUE #185: Stats: Total life playtime
See #38.

### PR DESCRIPTION
**Pull request by BatedUrGonnaDie**
Would close #185

### CODE REVIEW

File: app/assets/javascripts/segment_history_graph.coffee [hunk 1]
```diff
@@ -15,7 +15,7 @@ $ ->
   hidden_data = []
   first_non_null_attempt = Infinity
 
-  $.each gon.run.splits, (i, split) ->
+  $.each gon.run.raw_splits, (i, split) ->
```
Comment (glacials) [inline app/assets/javascripts/segment_history_graph.coffee]: What's this change for/do?

Comment (BatedUrGonnaDie) [inline app/assets/javascripts/segment_history_graph.coffee]: gon.run.raw_splits is supposed to have the complete splits with the history.  When I moved the full_splits loop in reset_graph.js it broke all of the splits due to trying to .filter a null.

Comment (glacials) [inline app/assets/javascripts/segment_history_graph.coffee]: I don't understand why it broke, from what I can tell the old `full_segments` code in StatsController produces the same result as the new `segments_with_history` code in CompletedRun, so the swap you did on StatsController line 24 (now 19) shouldn't cause anything that uses `gon.run.raw_splits` to notice a difference. Am I missing something?

Comment (BatedUrGonnaDie) [inline app/assets/javascripts/segment_history_graph.coffee]: I'm not exactly sure why it changes what populates gon.run.splits to be quite honest.  Checking the contents of it pre-commits shows that the history is present in gon.run.splits, but when the logic is moved out of the controller it no longer puts the history in gon.run.splits.  I'm not sure if there is a call to some function somewhere that I'm just missing.

Comment (glacials) [inline app/assets/javascripts/segment_history_graph.coffee]: ooo, yeah that's it :) nice catch! It was incorrect to begin with.

```
$ irb
>> b = [1, 2, 3]
=> [1, 2, 3]
>> a = b
=> [1, 2, 3]
>> a[0] = 2
=> 2
>> b
=> [2, 2, 3]
```

**Pull request merged**